data_processing/scanManager/generateScan.py

```diff
@@ -26,6 +26,9 @@
 @click.option('-s', '--container_id', required=True)
 @click.option('-m', '--method_id',    required=True)
 def cli(cohort_id, container_id, method_id):
+    container_generate_scan(cohort_id, container_id, method_id)
+
+def container_generate_scan(cohort_id, container_id, method_id):
```

Comment:
Click decorators change the import function resulting in `_pickle.PicklingError: Can't pickle <function  >: it's not the same object as`